docs/docs/reference/changed-features/pattern-matching.md

```diff
@@ -50,6 +50,13 @@ and `S` conforms to one of the following matches:
 The former form of `unapply` has higher precedence, and _single match_ has higher
 precedence over _name-based match_.
 
+A usage of a fixed-arity extractor is irrefutable if one of the following condition holds:
+
+- `U = true`
+- the extractor is used as a product match
+- `U = Some[T]` (for Scala2 compatibility)
+- `U <: R` and `U <: { def isEmpty: true }`
```

Comment:
Isn't it supposed to be `def isEmpty: false`?

Reply:
Yes, of course, good catch.